.gitignore

```diff
@@ -6,6 +6,9 @@ __pycache__/
 # C extensions
 *.so
 
+# Collect static files
+/static-root
```

Comment:
Why do you want to put static files in top system directory ?

Reply:
Done.